lib/protos.js

```diff
@@ -350,8 +342,10 @@ exports._wrapInitialize = function() {
 exports._wrapPage = function() {
   var page = this.page;
   this.page = function() {
-    if (this._assumesPageview && !this._initialized) {
-      return this.initialize.apply(this, arguments);
+    if (this._assumesPageview) {
```

Comment:
We could just have this be `this.page = after(2, this.page)` when `assumesPageView` is set, which will just noop the first call. Then we don't have to keep track of any additional state.